api/src/types/snapshot.ts

```diff
@@ -1,10 +1,12 @@
 import { Collection } from './collection';
 import { Relation, RelationMeta, Field, FieldMeta } from '@directus/shared/types';
 import { Diff } from 'deep-diff';
+import { DatabaseClient } from './database';
 
 export type Snapshot = {
 	version: number;
 	directus: string;
+	vendor?: DatabaseClient;
```

Comment:
Yeah, particularly because we also allow omitting the `vendor` flag via `force` query param, so old snapshots can still be used if it's their only option.

Reply:
Great, thanks! 😃 Makes sense 👍